yt/frontends/enzo/fields.py

```diff
@@ -44,6 +47,22 @@
     'AvgElec2': [1, 1, 0],
 }
 
+
+class OtherFieldInfo(pydantic.BaseModel):
```

Comment:
I'll look, yeah, and move `OtherFieldInfo`, as well. I made `display_name = ''` so that it did not need to be specified in the yaml file, since all of the fields for enzo had this attribute as `None`, so I thought yt would auto-use the code name. I can change it, though.